<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1104</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Overview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="OverviewTable">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>190</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="loadingLabel">
      <property name="text">
       <string>Loading candlestick data...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
      <item>
       <widget class="QGroupBox" name="SendOrderBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Submit Order</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="4" column="2">
              <widget class="QComboBox" name="SymbolComboBox"/>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="SizeLabel">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Size:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="PriceLabel">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Price:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="SymbolLabel">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Symbol:</string>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QSpinBox" name="Size">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>999999999</number>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QDoubleSpinBox" name="Price">
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="Symbol">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>60</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QPushButton" name="MarketBuyButton">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Market Buy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="MarketSellButton">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Markey Sell</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>5</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="LimitBuyButton">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Limit Buy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LimitSellButton">
               <property name="minimumSize">
                <size>
                 <width>125</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Limit Sell</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="PortfolioBox">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>My Portfolio</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="BuyingPowerLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100000</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Buying Power:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="BuyingPower">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100000</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>30</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="TotalShareLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100000</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Total Traded Shares:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TotalShare">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100000</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>30</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="TotalPLLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100000</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Total P&amp;L:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TotalPL">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100000</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="PortfolioTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="showGrid">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>190</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="OrderHistoryBox">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Waiting List</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableView" name="WaitingListTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>190</number>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="OrderIDLabel">
               <property name="text">
                <string>Order ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="CancelOrderID">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CancelSizeLabel">
               <property name="text">
                <string>Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="CancelOrderSize">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="CancelOrderButton">
               <property name="text">
                <string>Cancel Order</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="CancelAllButton">
               <property name="text">
                <string>Cancel All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOrderbook">
    <property name="title">
     <string>Order Book</string>
    </property>
    <addaction name="actionShow_Order_Book"/>
   </widget>
   <widget class="QMenu" name="menuCharts">
    <property name="title">
     <string>Charts</string>
    </property>
    <addaction name="actionCandlestick_Chart"/>
   </widget>
   <widget class="QMenu" name="menuThemes">
    <property name="title">
     <string>Preferences</string>
    </property>
    <widget class="QMenu" name="menuThemes_2">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="theme_none_action"/>
     <addaction name="theme_dark_action"/>
     <addaction name="theme_material_action"/>
    </widget>
    <addaction name="menuThemes_2"/>
   </widget>
   <addaction name="menuOrderbook"/>
   <addaction name="menuCharts"/>
   <addaction name="menuThemes"/>
  </widget>
  <action name="actionShow_Chart">
   <property name="text">
    <string>Show Candlestick Chart</string>
   </property>
  </action>
  <action name="actionShow_Order_Book">
   <property name="text">
    <string>Show Order Book</string>
   </property>
  </action>
  <action name="actionCandlestick_Chart">
   <property name="text">
    <string>Candlestick Chart</string>
   </property>
  </action>
  <action name="theme_none_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="theme_dark_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SHIFT Dark</string>
   </property>
  </action>
  <action name="theme_material_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Material</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>CancelOrderButton</tabstop>
  <tabstop>CancelAllButton</tabstop>
  <tabstop>PortfolioTable</tabstop>
  <tabstop>OverviewTable</tabstop>
  <tabstop>WaitingListTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
